shared/src/components/Header/Header.test.tsx

```diff
@@ -149,7 +157,7 @@ describe("Header", () => {
         logout={jest.fn()}
       />
     );
-    const currentMenuItem = screen.getByRole("link", { current: "page" });
+    const currentMenuItem = screen.getAllByRole("link", { current: "page" })[0];
```

Comment:
Some navigation items appear in a menu at middle screen sizes. The display of that menu is controlled by media queries so that at certain sizes the nav links are hidden and the menu is shown. This means that a number of links exist in the DOM twice (but only one is visible at a time) and have the current page prop set.

Reply:
Yeah I had a similar issue somewhere else. Unless we use media queries in JS I don't see a way around this unfortunately.

Reply:
@huwshimi @Caleb-Ellis  Sorry, I missed this.

This test passes when changing to single item query `getByRole` as I suggested, which means that the current page link must only exist once. I think we might have bug here. Will look into this as part of https://github.com/canonical-web-and-design/app-tribe/issues/862

Reply:
Looks like links only ever appear once until you open a dropdown -  we're adding aria-hidden when it's closed (and items within are correctly ignored by testing-library).